src/components/container/ReasonCard/ReasonCard.scss

```diff
@@ -0,0 +1,53 @@
+.ReasonCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 16px;
+  max-width: 356px;
+  font-style: normal;
+
+  &__icon {
+    position: relative;
+    width: 48px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+
+    &__background {
+      width: 100%;
+      height: 100%;
+      background-color: var(--color-selective-yellow);
+      border-radius: 16px;
+      position: absolute;
+      left: 0%;
+      right: 0%;
+      top: 0%;
+      bottom: 0%;
+
+      &__image {
+      position: absolute;
+      left: 25%;
+      right: 25%;
+      top: 25%;
+      bottom: 25%;
+    }
+    }
+  }
+
+  &__title {
+    color: var(--color-bunting);
```

Comment:
we might want to remove default margins of h3 element, because per design there isn't. and gap property is responsible for the spaces between the flex children

Reply:
I also removed the margins in my component, but wouldn't it make more sense to just do that in the global CSS? Also the styling of the blue ribbon for h2::after elements to match the figma template

@jasper-prgh Since the global styles ticket is already done, would it maybe make sense to add a new one for these things?

Reply:
@timmy38233 yes. absolutely